ecosystem_tests/ecosystem_tests_cli/commands/blueprint_linting.py

```diff
@@ -0,0 +1,132 @@
+########
+# Copyright (c) 2014-2022 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import subprocess
+from git import Repo
+from github import Github
+from ..logger import logger
+from datetime import datetime
+from ...ecosystem_tests_cli import ecosystem_tests
+from ecosystem_cicd_tools.new_cicd.github import with_github_client
+
+FILE_TYPE = ".yaml"
+BLUEPRINT_START = "tosca_definitions_version:"
```

Comment:
It's cleaner to put format strings like command templates in the top of the file as a constant.

```suggestion
BLUEPRINT_START = "tosca_definitions_version:"
LINT_CMD = "cfy-lint -b {} --format JSON"
AF_CMD = "cfy-lint -b {} -af"
```